### PR TITLE
add tikv label merge policy description

### PR DESCRIPTION
**Pull request by pepezzzz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -1281,6 +1281,18 @@ Encoding 格式示例：
...
+
+{{< copyable "" >}}
+
+```bash
+>> store label 1 --force
+```
+
+> **注意：**
+>
+> * store 的 label 方法使用合并策略，除 PD 可以存放 store label 信息外，TiKV 进程启动配置文件内也可以存放 store label 信息，两者会进行合并标识。建议不要同时使用两种 label 方式，使用 tiup 修改配置文件的 label 时，PD 存放的 store label 信息可以使用 store label <id> --force 进行清空，实现 store label 的集中管理。
```
Comment (Oreoxmt) [inline pd-control.md:1294]:
```suggestion
> - store 的 label 方法使用合并策略，除 PD 可以存放 store label 信息外，TiKV 进程启动的配置文件内也可以存放 store label 信息，两者会进行合并标识。
> - 建议不要同时使用两种 label 存放方式。使用 `tiup` 修改配置文件的 label 时，PD 存放的 store label 信息可以使用 `store label <id> --force` 进行清空，实现 store label 的集中管理。
```

File: pd-control.md [hunk 2]
```diff
@@ -1281,6 +1281,19 @@ Encoding 格式示例：
...
+
+{{< copyable "" >}}
+
+```bash
+>> store label 1 --force
+```
+
+> **注意：**
+>
+> - store 的 label 方法使用合并策略，除 PD 可以存放 store label 信息外，TiKV 进程启动的配置文件内也可以存放 store label 信息，两者会进行合并标识。
```
Comment (rleungx) [inline pd-control.md:1294]: How about
```suggestion
> - store 的 label 更新方法使用的是合并策略。如果修改了 TiKV 配置文件中的 store label，进程重启之后，PD 会将自身存储的 store label 与其进行合并更新，并持久化合并后的结果。
> - 如果希望使用 TiUP 统一管理 store label 的话，可以在集群重启前，使用 pd-ctl 的 `store label <id> --force` 命令将 PD 存储的 store label 清空。
```

Comment (pepezzzz) [reply pd-control.md:1294]: 并不会 “与其进行合并更新，并持久化合并后的结果。” 两个位置的配置还是独立的，只是会 merge。

Comment (rleungx) [reply pd-control.md:1294]: 实际调度生效配置都是以 PD 为准的，只是更新这个配置可以理解为有两个入口，一个是通过 PD 的 API，一个是通过 TiKV 修改配置文件。这两者都会与 PD 中的 label 信息进行一个 merge，并最终更新到 store 的元信息里面持久化到 etcd，后续调度都是以 store 元信息里面的 label 为准。

File: pd-control.md [hunk 3]
```diff
@@ -1281,6 +1281,20 @@ Encoding 格式示例：
...
+
+```bash
+>> store label 1 --force
+```
+
+> **注意：**
+>
+> - store 的 label 更新方法使用的是合并策略。如果修改了 TiKV 配置文件中的 store label，进程重启之后，PD 会将自身存储的 store label 与其进行合并更新，并持久化合并后的结果。
+> - 如果希望使用 TiUP 统一管理 store label 的话，可以在集群重启前，使用 pd-ctl 的 `store label <id> --force` 命令将 PD 存储的 store label 清空。
+> - 建议不要同时使用两种 label 存放方式。使用 `tiup` 修改配置文件的 label 时，PD 存放的 store label 信息可以使用 `store label <id> --force` 进行清空，实现 store label 的集中管理。
```
Comment (rleungx) [inline pd-control.md:1296]: 这一条要不要删掉

Comment (Oreoxmt) [inline pd-control.md:1296]:
```suggestion
```

**Review by Oreoxmt — APPROVED** [review]

**Review by rleungx — APPROVED** [review]

File: pd-control.md [hunk 4]
```diff
@@ -1281,6 +1281,19 @@ Encoding 格式示例：
...
+{{< copyable "" >}}
+
+```bash
+>> store label 1 --force
+```
+
+> **注意：**
+>
+> - store 的 label 更新方法使用的是合并策略。如果修改了 TiKV 配置文件中的 store label，进程重启之后，PD 会将自身存储的 store label 与其进行合并更新，并持久化合并后的结果。
+> - 如果希望使用 TiUP 统一管理 store label 的话，可以在集群重启前，使用 pd-ctl 的 `store label <id> --force` 命令将 PD 存储的 store label 清空。
```
Comment (TomShawn) [inline pd-control.md:1295]:
```suggestion
> - 如果希望使用 TiUP 统一管理 store label 的话，可以在集群重启前，使用 PD Control 的 `store label <id> --force` 命令将 PD 存储的 store label 清空。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**